.env.template

```
@@ -1,3 +1,13 @@
+VITE_ENVIRONMENT=
```

Maybe call it like this to make it clear that it’s used in Sentry?
```suggestion
VITE_SENTRY_ENVIRONMENT=
```